webapp/app/translations/de/LC_MESSAGES/messages.po

```diff
@@ -4546,7 +4546,10 @@ msgid "info-stl.card-partner.p1-content"
 msgstr ""
 "Ja. Wenn Sie verheiratet sind oder in einer eingetragenen Partnerschaft "
 "leben, können Sie Ihre Steuererklärung gemeinsam als Paar machen. Es "
-"reicht in diesem Fall aus, wenn sich eine Person registriert."
+"reicht in diesem Fall aus, wenn sich nur eine Person registriert und "
+"unter \"Ihre Steuererklärung\" für die gemeinsame Steuererklärung "
+"anmeldet. Im Steuerformular können Sie dann zusätzlich zu Ihren Angaben "
```

Comment:
I have seen it in the ticket, but in my opinion a comma there would be wrong

Reply:
You are right, it sounds quite weird. Then is ok for me

Reply:
It is possible to set two  _Im Steuerformular können Sie dann, zusätzlich zu Ihren Angaben, 
auch die Daten zur Ihrem Partner oder ihrer Partnerin eintragen._ This would be grammatically correct.